test/integration/patients/list/patients-all.js

```diff
@@ -34,6 +53,24 @@ context('patient sidebar', function() {
       .visit('/patients/all')
       .wait('@routePatients');
 
+    cy
+      .get('.app-frame__content')
+      .find('.table-list__item')
+      .first()
+      .should('contain', 'First Last');
+
+    cy
+      .get('.button--blue')
+      .should('contain', 'All Groups')
+      .click();
+
+    cy
```

Comment:
actually this part of the test doesn't mean anything and should be removed.  The stub can return anything it wants and doesn't care what the filter is set to.  The way to test this will be by seeing if the endpoint is hit with the correct filter data.  So rather than setup the test to look like we're testing the group data base on the filter selection, let's avoid doing that and just test the data from the stub.. then later test the filter data... with nothing list related once the filter is selected.